### ISSUE TITLE
Request: Stalker Portal Support

### ISSUE THREAD

**Issue by blgjvc** [linuxmint/hypnotix#173]
Is there any way to add Stalker Portal support (IPTV list with url and mac address)?

**Comment by DaBulgarian** [comment]
Yes adding a Stalker Portal support would be a killer feature! I know so many people are looking for it, so if you guys can add it, you will be golden!

**Comment by sam5558** [comment]
:+1:
